### DISCUSSION TITLE
Synced Google / Caldav unreadable in widget

### DISCUSSION THREAD

**Discussion by rafnov** [FossifyOrg/General-Discussion#464] [Q&A]
How to make them readable?![Screenshot_2024-09-08-08-09-22-61_b783bf344239542886fee7b48fa4b892.jpg](https://github.com/user-attachments/assets/7dd50989-84ea-428b-b5fb-93501b826a23)

Why those events synced with Google calendar local account look like this? Unreadable.
How to display them correctly?

**Comment by Aga-C** [comment]
1. Have you tried creating the widget once again?
2. In Settings → `Manage event types` you can change the color of the calendar. Does it help?

**Reply by rafnov** [reply]
#2 helped, thx.
